### PR TITLE
2nd part

### PR DESCRIPTION
**Pull request by David-Verde**
Add API link
Create API functionality 
create JS modules

### CODE REVIEW

File: src/newGame.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ name: 'Digitalization' }),
+  });
+  const data = await res.json();
+  gameId = (data.result).split(' ');
+  return gameId[3];
+}
```
Comment (youmari) [inline src/newGame.js]: - [ ] please try to use the `arrow function` instead of `function keyword` in order to practice working with both

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @David-Verde , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
✔️ Great title and short summary 

✔️ Good commit messages
✔️ No linter error

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by FrederickMih — COMMENTED** [review]
Hi @David-Verde , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project with some failing Linters checks. Let's fix it and submit the project for review once more.
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢🟢🟢 

Hi, @David-Verde,

YesYouAreGIF

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights
✅ Correct use of Gitflow
✅ No linter error
✅ README is professional
✅ App works fine 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**